### DISCUSSION TITLE
正しくリマインドされることを確認するテスト

### DISCUSSION THREAD

**Discussion by chikowaka** [ushmz/subsuke#32] [Product backlog]
誰が　　　：
何をしたい：

テスト要件
前提条件　：
手順　　　：
入力値　　：
期待する値：

**Comment by ushmz** [comment]
特定の時刻とUI上で（表示されるかどうかの確認）のテストが必要になる？

**Comment by ushmz** [comment]
https://github.com/ushmz/subsuke/blob/a8c5880c62620c60592015c392c7301a447e43c1/lib/notifications/notifications.dart#L55-L84
